SECURITY.md

```diff
@@ -0,0 +1,18 @@
+# Security Policy
+
+If you have discovered a security vulnerability in this project, please report it
+privately. **Do not disclose it as a public issue.** This gives us time to work with you
+to fix the issue before public exposure, reducing the chance that the exploit will be
+used before a patch is released.
+
+You may submit the report in the following ways:
+
+- send an email to ???@???; and/or
+- send us a [private vulnerability report](https://github.com/AOMediaCodec/libavif/security/advisories/new)
+
+Please provide the following information in your report:
+
+- A description of the vulnerability and its impact
```

Comment:
I would include the version tested or the output from `git describe`.

Reply:
Looking at the docs a little more closely, maybe that's a little redundant if the recommendation is the reporter start a private fork to coordinate.

Reply:
A private fork seems too much work just to report something. Reproducing is the most important part, so version and instructions are enough. A description of the vulnerability and its impact are nice to have, but it should be optional. Unless we expect from reporters to justify why they did not file just a simple bug; but the consequence might be that they file it publicly or do not file it at all.